packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -122,33 +124,33 @@ Create the Okta event hook to work with your local application, which can now be
 
 8. Click **Save & Continue**.
 
-9. With your ngrok session and local application running, complete the one-time verification Okta call at this time. You can also verify the event hook later.
+9. With your ngrok session and local application running, complete the one-time verification Okta call now. You can also verify the event hook later.
 
 ### Verify the event hook
 
-You must verify the event hook to prove that your external service controls the endpoint. See [One-Time Verification Request](/docs/concepts/event-hooks/#one-time-verification-request).
+Verify the event hook to prove that your external service controls the endpoint. See [One-Time Verification Request](/docs/concepts/event-hooks/#one-time-verification-request).
 
 To complete the one-time verification of the event hook:
 
 * After creating the event hook, and:
 
   * Your local application server is running (`>node server.js`).
 
-  * Your local ngrok session is running (and the forwarding URL is the same as the URL field defined in the Event hook).
+  * Your local ngrok session is running (and the forwarding URL is the same as the URL field defined in the event hook).
 
   Click **Verify** to complete the one-time verification step.
 
-or
+Or:
 
-* After making sure that your  ngrok session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** dropdown menu of your **UNVERIFIED** event hook, and select **Verify**.
+* After making sure that your ngrok session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** dropdown menu of your **UNVERIFIED** event hook, and select **Verify**.
 
 The event hook is now set up with a status of **VERIFIED** and is ready to send event hook calls to your external service.
 
->**Note:** A successful event hook verification also indicates your local application is working with the ngrok session! Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.
+>**Note:** A successful event hook verification also indicates that your local application is useable with the ngrok session. Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.
```

Comment:
```suggestion
>**Note:** A successful event hook verification also indicates that your local application is usable with the ngrok session. Review the ngrok terminal or inspector interface for details on the first `GET` call to your local application.
```

Reply:
Is this a Canada/US spelling difference?

Reply:
Not sure .... MMS uses your suggestion.